modules/bmo-configuring-the-networkdata-parameter-in-the-bmo-cr.adoc

```diff
@@ -0,0 +1,54 @@
+// This module is included in the following assemblies:
+//
+// * installing/installing_bare_metal/bare-metal-using-bare-metal-as-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="bmo-configuring-the-networkdata-parameter-in-the-bmo-cr_{context}"]
+= Configuring the networkData parameter in the BareMetalHost resource
+
+The `networkData` field in the `BareMetalHost` custom resource (CR) allows you to control the network configuration of the bare-metal host at creation time. For most operating systems, this is achieved using a configuration file encapsulated in a Kubernetes secret. Then, the `cloud-init` service uses it to customize services.
+
+.Procedure
+
+. Create a file named `network-data.yaml` with the following content:
++
+[source,yaml]
+----
+links:
+  - id: <interface_id>
+    type: phy
+    ethernet_mac_address: <mac_address>
+networks:
+  - id: <interface_id>
+    link: <interface_id>
+    type: ipv4_dhcp
+services:
+  - type: dns
+    address: <dns_server>
+----
++
```

Comment:
Not a merge blocker, but definition lists that define variables should be prefixed with `where:` and the definition begins with "Specifies". See the [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks) (third bullet). This applies throughout but is not urgent.